assignments/callbacks.js

```diff
@@ -27,29 +27,65 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr);
 }
 
+getLength(items, function(first) {
+  console.log(first)
+});
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[3]);
```

Comment:
How would you get the last item in an array? You could do something like `array[array.length -1]`.